core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1084,7 +1080,9 @@ private void deleteAndJournal(LockedInodePath inodePath, DeleteOptions deleteOpt
     long opTimeMs = System.currentTimeMillis();
     deleteInternal(inodePath, false, opTimeMs, deleteOptions);
     DeleteFileEntry deleteFile = DeleteFileEntry.newBuilder().setId(fileId)
-        .setRecursive(deleteOptions.isRecursive()).setOpTimeMs(opTimeMs).build();
```

Comment:
yes, that looks like a bug ... could you please add a unit test for this? thanks